xml/System.Net.Quic/QuicListenerOptions.xml

```diff
@@ -140,7 +141,7 @@
       <Docs>
         <summary>Gets or sets the endpoint to listen on.</summary>
         <value>To be added.</value>
-        <remarks>This property is mandatory, and not setting it will result in validation errors when starting the listener.</remarks>
+        <remarks>This property is mandatory, not setting will result in validation errors when starting the listener.</remarks>
```

Comment:
```suggestion
        <remarks>This property is mandatory. Not setting it will result in validation errors when starting the listener.</remarks>
```